src/locales/en.json

```diff
@@ -130,42 +130,43 @@
       "audio": "Audio Sources"
     }
   },
-  "meta-search-page": {
-    "title": "Meta Search",
+  "external-sources-page": {
+    "title": "External Sources",
     "intro": "{openverse} is built on top of a catalog that indexes CC-licensed and public domain content from selected sources. Learn more about our {link}.",
-    "link": "sources",
-    "license": "The internet is a big place! There are plenty of search portals and collections, hosting CC-licensed and public domain content, which we aren’t able to index or surface results from within the {openverse} interface. These are important sources of CC-licensed and public domain content, and we want to make sure that you are able to find the best openly licensed materials possible, regardless of where they are located.",
-    "content": "What content types can I search for using the Meta Search feature?",
-    "content-types": "Meta Search on {openverse} currently supports image, audio, and video search on external sites.",
+    "link": "sources here",
+    "license": "However, there are many sources of CC-licensed and public domain media which we aren’t able to surface results from within the {openverse} interface yet. This might be because they do not offer a public API, or that our contributors have not yet had time to integrate them into Openverse. These are valued sources and we want to make sure that you are able to find the best openly licensed materials possible, regardless of where they are located.",
     "use": " Use ",
     "images": {
-      "title": "How does the Meta Search feature work for indexed media types?",
-      "demo-label": "Layout of Meta Search for indexed media type, like images",
+      "title": "How does the External Sources feature work for indexed media types?",
+      "demo-label": "Layout of External Sources for indexed media type, like images",
```

Comment:
Since "External Sources" is so generic (unlike "Meta Search" which was a proper feature name) maybe we should quote it when using in sentences.